### ISSUE TITLE
What is the recommended way to handle imbalanced data?

### ISSUE THREAD

**Issue by DieseKartoffel** [ck37/coral-ordinal#8]
Hello,

i hope this is the right place to ask this.

I have a dataset of ordinal data where the classes significantly decrease in frequency, similar to older ages being less common in a large population. How would you adress this in a coral-ordinal model optimally? 

Is using `compute_class_weight` from sklearn.utils.class_weight on the one hot encoded labels a valid approach? How would you pass the results to the class_weight parameter of the keras model.fit() function?

Thank you!

**Comment by Omar3esam** [comment]
You can use 
```python
class_weight
```
Argument in .fit method
